src/reporting.js

```diff
@@ -330,11 +330,15 @@ async function generateReport(console, httpUrl, matrixFile, filePath, tempDir, s
         }
 
         // Further validate the mimetype of the file from the decrypted content
-        let mimetype = fileType(decryptedFileContents);
-        if (mimetype === null) {
-            return {clean: false, info: 'File type not supported'};
-        } else if (mimetypeArray && !mimetypeArray.includes(mimetype)) {
-            return {clean: false, info: 'File type not supported'};
+        if (mimetypeArray) {
+            const mimetype = fileType(decryptedFileContents);
+            if (mimetype === null) {
+                console.info(`Skipping unsupported decrypted file - unknown mimetype`);
```

Comment:
We may want to always the filePath or some other identifier as this method is async and may be called multiple times simultaneously, thus jumbling up the logs?

Reply:
Good idea, added!

Reply:
Thanks!

I'm a bit on the fence about whether the end of a log line is the best place for it. Although I know the filepath might be long, it may also be difficult to parse multiple log lines when the filepath is at different character column depths.

We already log the full filepath earlier in execution, so there's not much point in logging whole thing again. Perhaps we can just log the filename and that would be less noisy to put at the front of the log line?

One can get just the filename from a fullpath with:

```nodejs
path.basename(filePath)
```

Reply:
I suppose I was thinking it as an easy to follow identifier across multiple lines, being in the same place and also the same identifier for the decrypted file errors, to avoid linking those together.

I think the `basename` will be useless, since the temporary file is not unique afaict:

`const filePath = path.join(tempDir, 'downloadedFile')`

I'll think of something and try improve, maybe a request identifier for all log lines?

Reply:
Ah, I see. It's a temporary directory, not filename.

Request identifier sounds more sensible, yeah. And would probably be best to do in a separate PR.